libs/map-editor/src/types.ts

```diff
@@ -123,7 +123,15 @@ export const RestrictedRightsPropertyData = PropertyBase.extend({
     type: z.literal("restrictedRightsPropertyData"),
     writeTags: z.array(z.string()).default([]),
     readTags: z.array(z.string()).default([]),
-    searchable: z.boolean().default(false),
+});
+
+export const PersonalAreaAccessClaimMode = z.enum(["dynamic", "static"]);
+
+export const PersonalAreaPropertyData = PropertyBase.extend({
+    type: z.literal("personalAreaPropertyData"),
+    accessClaimMode: PersonalAreaAccessClaimMode,
+    allowedTags: z.array(z.string()).default([]),
+    ownerId: z.string().default(""),
 });
```

Comment:
Not really sure about that. The claim mode is changed by the user. OwnerID is applied when personal area is assigned/claim to/by a user. I think adding 2 differents types will add complexity in UI, typings etc. However it will be relevant if dynamic area has more than 1 field in the type.